### ISSUE TITLE
Add tests for unsafe paste detection

### ISSUE THREAD

**Issue by danirabbit** [elementary/terminal#801]
### Problem

As unsafe paste detection gets more complex in the types of commands it checks for, the chance for regressing here increases

### Proposal

We should make sure to have tests so that it's clear what we're testing for and that we don't regress on detecting those things

**Comment by teamcons** [comment]
This would be my first thought (as vala newbie)
-Move the if/or tapdancing into actual full fledged check_if_multiple, check_if_admin functions who manage all the logic (take a string, spit a boolean)
-test how well those functions works again several test strings
-Id be also fine doing testing on those functions (like against our bash history to see what those functions, as standalone binaries, can flag, or on the daily usage)

**Comment by jeremypw** [comment]
Yes, some refactoring along those lines is probably required now the checks are more extensive.

**Comment by teamcons** [comment]
Yall could have simply told me to put some more effort in my PR you wet sandwiches :)

**Comment by danirabbit** [comment]
Fixed in #818
